### DISCUSSION TITLE
Separate Inventory and InventoryGUI

### DISCUSSION THREAD

**Discussion by Balowser** [peter-kish/gloot#213] [General]
Hey,
I try to separate my GUI and the Inventory.
I attached the Inventory to my Player-Scene and the Inventory-Control to my Main GUI-Node.

In the GUI-Node I need a reference to my Player-Inventory. But with the "three points" I can not add another node, only nodes in the same Scene as the GUI.
How Can I solve this?

**Comment by peter-kish** [comment]
Hi!
You should be able to do that from GDScript in the scene that contains both, the player scene and the GUI scene. The `inventory` property of the inventory controls can be used for that. E.g.:
```gdscript
@onready var ctrl_inventory = $GUI/CtrlInventory
@onready var player = $Player

func _ready() -> void:
    ctrl_inventory.inventory = player.inventory
```

**Reply by Balowser** [reply]
Oh okay, I will try it later.

Thanks!
